sys/arm64/include/ucontext.h

```diff
@@ -71,18 +71,43 @@ struct __mcontext {
 	struct fpregs	mc_fpregs;
 	int		mc_flags;
 #define	_MC_FP_VALID	0x1		/* Set when mc_fpregs has valid data */
+#define	_MC_CAP_VALID	(1u<<31)	/* Set when mc_capregs has valid data */
```

Comment:
I've changed this from morello-dev so it won't conflict if upstream adds flags, e.g. for SVE.